--- a/Windows/src/pb_internal.h
+++ b/Windows/src/pb_internal.h
@@ -23,7 +23,7 @@
 #define MAXBUF 0xFFFF
 #define LOCAL_PROXY_PORT 34010
 #define LOCAL_UDP_RELAY_PORT 34011  // its running UDP port still make sure to not run on same port as TCP, opening same port and tcp and udp cause issue and handling port at relay server response injection
-#define MAX_PROCESS_NAME 1024
+#define MAX_PROCESS_NAME 65536
 #define VERSION "4.0.13-Beta"
 #define PID_CACHE_SIZE 1024
 #define PID_CACHE_TTL_MS 30000
